packages/rpc/README.md

```diff
@@ -1,93 +1,73 @@
+		<a href="https://github.com/discordjs/discord.js/actions"><img src="https://github.com/discordjs/discord.js/actions/workflows/test.yml/badge.svg" alt="Tests status" /></a>
+		<a href="https://codecov.io/gh/discordjs/discord.js" ><img src="https://codecov.io/gh/discordjs/discord.js/branch/main/graph/badge.svg?precision=2&flag=rpc" alt="Code coverage" /></a>
+	</p>
+	<p>
+		<a href="https://vercel.com/?utm_source=discordjs&utm_campaign=oss"><img src="https://raw.githubusercontent.com/discordjs/discord.js/main/.github/powered-by-vercel.svg" alt="Vercel" /></a>
+	</p>
 </div>
 
 ## Installation
 
 **Node.js 16.9.0 or newer is required.**
 
 ```sh-session
-npm install @discordjs/rest
-yarn add @discordjs/rest
-pnpm add @discordjs/rest
+npm install @discordjs/rpc
+yarn add @discordjs/rpc
+pnpm add @discordjs/rpc
 ```
 
 ## Examples
 
-Install all required dependencies:
-
-```sh-session
-npm install @discordjs/rest discord-api-types
-yarn add @discordjs/rest discord-api-types
-pnpm add @discordjs/rest discord-api-types
-```
-
-Send a basic message:
+Rich presence example
 
 ```js
-import { REST } from '@discordjs/rest';
-import { Routes } from 'discord-api-types/v10';
+import { Client } from '@discordjs/rpc';
 
-const rest = new REST({ version: '10' }).setToken('token');
+const clientId = '287406016902594560';
 
-try {
-	await rest.post(Routes.channelMessages(CHANNEL_ID), {
-		body: {
-			content: 'A message via REST!',
-		},
-	});
-} catch (error) {
-	console.error(error);
-}
-```
+const client = new Client({ transport: 'ipc' });
 
-Create a thread from an existing message to be archived after 60 minutes of inactivity:
-
-```js
-import { REST } from '@discordjs/rest';
-import { Routes } from 'discord-api-types/v10';
-
-const rest = new REST({ version: '10' }).setToken('token');
-
-try {
-	await rest.post(Routes.threads(CHANNEL_ID, MESSAGE_ID), {
-		body: {
-			name: 'Thread',
-			auto_archive_duration: 60,
-		},
+client.on('ready', () => {
+	client.setActivity({
+		details: 'i am snek',
+		state: 'in slither party',
+		largeImageKey: 'snek_large',
+		largeImageText: 'tea is delicious',
+		smallImageKey: 'snek_small',
+		smallImageText: 'i am my own pillows',
+		instance: false,
 	});
-} catch (error) {
-	console.error(error);
-}
+});
+
+client.login({ clientId });
 ```
 
 ## Links
 
 - [Website](https://discord.js.org/) ([source](https://github.com/discordjs/discord.js/tree/main/packages/website))
-- [Documentation](https://discord.js.org/#/docs/rest)
+- [Documentation](https://discord.js.org/#/docs/rpc)
 - [Guide](https://discordjs.guide/) ([source](https://github.com/discordjs/guide))
-  See also the [Update Guide](https://discordjs.guide/additional-info/changes-in-v13.html), including updated and removed items in the library.
+  See also the [Update Guide](https://discordjs.guide/additional-info/changes-in-v14.html), including updated and removed items in the library.
```

Comment:
The update guide does not has rpc information so this is somewhat redunant

Reply:
we are applying changes from upstream / other packages